test/test_matrix4x4.py

```diff
@@ -312,3 +312,18 @@ def test_implicit_convert_to_numpy():
     pose2 = zivid.calibration.Pose(numpy.array(sample))
 
     assert_all_equal_2d(zivid.Matrix4x4(pose1.to_matrix()), pose2.to_matrix())
+
+
+def test_identity():
+    identity = zivid.Matrix4x4.identity()
+
+    for row in range(4):
+        for col in range(4):
+            if row == col:
+                numpy.testing.assert_almost_equal(
+                    identity[row, col], 1, ZIVID_MATRIX_SAVE_LOAD_TOLERANCE_DECIMAL
+                )
+            else:
+                numpy.testing.assert_almost_equal(
+                    identity[row, col], 0, ZIVID_MATRIX_SAVE_LOAD_TOLERANCE_DECIMAL
+                )
```

Comment:
Can maybe just do
```
identity = zivid.Matrix4x4.identity()
assert identity == np.eye(4)
```

Also I think we should do exact comparison here, i.e. zero tolerance.